hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterServiceImpl.java

```diff
@@ -693,7 +694,7 @@ public boolean isJoined() {
         return joined.get().isJoined;
     }
 
-    @Probe(name = CLUSTER_METRIC_CLUSTER_SERVICE_SIZE)
+    @Probe(name = CLUSTER_METRIC_CLUSTER_SERVICE_SIZE, level = DEBUG)
```

Comment:
We ignore this because under normal operations we send the same value periodically?

Reply:
It's fine for me either way too. It just looked a bit different from the other ones we exclude, so I asked :smile:

Reply:
this change stood out for me too and it's quite a basic metric (cluster size) so I'd rather revert this change